src/library/Box/EventManager.php

```diff
@@ -38,9 +38,7 @@ public function fire($data)
         $subject = $data['subject'] ?? null;
         $params = $data['params'] ?? null;
 
-        if (BB_DEBUG) {
-            $this->di['logger']->debug($event.': '.var_export($params, 1));
-        }
```

Comment:
Are we always logging debug events now? Or again, has this check been moved? 🙂

Reply:
There already is a check here, it's redundant:

https://github.com/FOSSBilling/FOSSBilling/blob/df4dba4c77d917e482fdaaecf40289f96414dd82/src/library/Box/Log.php#L138-L141

Reply:
Great - thanks - just what I was looking for! 😄